gci/config.py

```diff
@@ -15,6 +16,7 @@ class TokenMissing(KeyError):
 
 
 def get_api_key(name):
+    logger = logging.getLogger(__name__ + 'get_api_key')
```

Comment:
logging names are dot separated.  so this should be `__name__ + '.get_api_key'` .  The same applies everywhere in your patch.

Reply:
ok